ndcube/asdf/converters/tablecoord_converter.py

```diff
@@ -0,0 +1,85 @@
+from asdf.extension import Converter
+
+
+class TimeTableCoordConverter(Converter):
+    tags = ["tag:sunpy.org:ndcube/extra_coords/table_coord/timetablecoordinate-*"]
+    types = ["ndcube.extra_coords.table_coord.TimeTableCoordinate"]
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from ndcube.extra_coords.table_coord import TimeTableCoordinate
+
+        table = node.get("table")
+        names = node.get("names")
+        physical_types = node.get("physical_types")
+        reference_time = node.get("reference_time")
+        timetablecoordinate = TimeTableCoordinate(
+            table, names=names, physical_types=physical_types, reference_time=reference_time)
```

Comment:
```suggestion
            node["table"], names=names, physical_types=physical_types, reference_time=reference_time)
```
Since table is required by the schema we can be sure it exists and don't need the `get`.

Reply:
Sure